src/unix/internal.h

```diff
@@ -396,35 +396,18 @@ UV_UNUSED(static char* uv__basename_r(const char* path)) {
   return s + 1;
 }
 
-UV_UNUSED(static int uv__fstat(int fd, struct stat* s)) {
-  int rc;
-
-  rc = fstat(fd, s);
-  if (rc >= 0)
-    uv__msan_unpoison(s, sizeof(*s));
-
-  return rc;
+#define UV_STAT_DEFINE(syscall, type)                                          \
+UV_UNUSED(static int uv__##syscall(type arg, struct stat* s)) {                \
+    int rc;                                                                    \
+    rc = syscall(arg, s);                                                      \
+    if (rc >= 0)                                                               \
+      uv__msan_unpoison(s, sizeof(*s));                                        \
+    return rc;                                                                 \
 }
```

Comment:
```suggestion
  int rc;                                                                      \
  rc = syscall(arg, s);                                                        \
  if (rc >= 0)                                                                 \
    uv__msan_unpoison(s, sizeof(*s));                                          \
  return rc;                                                                   \
}
```

Reply:
Right! Fixed in https://github.com/libuv/libuv/pull/4500/commits/c827352c182cfce5a2fa4331a747d136ed731dc5